example/ios/Flutter/AppFrameworkInfo.plist

```diff
@@ -21,6 +21,6 @@
   <key>CFBundleVersion</key>
   <string>1.0</string>
   <key>MinimumOSVersion</key>
-  <string>8.0</string>
+  <string>9.0</string>
```

Comment:
Any affect on this version change?

Reply:
This change related to I update the flutter version to the latest one which is v2.5.3. It's auto-updated the MiniumnOSVersion from 8.0 to 9.0 when you run `flutter build ios`. 
I don't think this version update will affect anything. I also tried to convert the version back stable v2.0.0(the version start to support flutter [web),](https://medium.com/flutter/whats-new-in-flutter-2-0-fe8e95ecc65) everything works fine and this change won't be included.